fs/fs.go

```diff
@@ -494,10 +495,16 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 		NegativeTimeout: &fs.negativeTimeout,
 		NullPermissions: true,
 	})
+	// Pass in a logger to go-fuse with the layer digest
+	// The go-fuse logs are useful for tracing exactly what's happening at the fuse level.
+	logger := log.L.
```

Comment:
Sanity check, we want to use the standard logger here instead of the context logger?

Reply:
I think so. This logger is attached to the context of the layer, not the context of the Mount operation (specifically, I don't think we care about the mountpoint in the fuse logs).